src/h5web/visualizations/shared/utils.ts

```diff
@@ -102,12 +109,15 @@ export function getValueToIndexScale(
   return valueToIndexScale;
 }
 
-export function getAxisScale(info: AxisInfo, rangeSize: number): AxisScale {
-  const { scaleFn, domain } = info;
+export function getCanvasScale(
```

Comment:
I felt this renaming was needed: the scale computed by this function relates the data values to the **canvas** coordinates.

The scale of `Axis` are different as they map the data values to the axis length,